volume-providers/vpc/vpcclient/instances/iks_get_volume_attachment.go

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Cloud Container Service, 5737-D43
+ * (C) Copyright IBM Corp. 2018, 2019 All Rights Reserved.
+ * The source code for this program is not  published or otherwise divested of
+ * its trade secrets, irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+
+package instances
+
+import (
+	"github.com/IBM/ibmcloud-storage-volume-lib/lib/utils"
+	"github.com/IBM/ibmcloud-storage-volume-lib/volume-providers/vpc/vpcclient/client"
+	"github.com/IBM/ibmcloud-storage-volume-lib/volume-providers/vpc/vpcclient/models"
+	"go.uber.org/zap"
+	"time"
+)
+
+// GetVolumeAttachment retrives the volume attach status with given volume attachment details
+func (vs *IKSVolumeAttachService) GetVolumeAttachment(volumeAttachmentTemplate *models.VolumeAttachment, ctxLogger *zap.Logger) (*models.VolumeAttachment, error) {
+	defer util.TimeTracker("IKS GetVolumeAttachment", time.Now())
+
+	operation := &client.Operation{
+		Name:        "GetVolumeAttachment",
+		Method:      "GET",
+		PathPattern: vs.pathPrefix + instanceIDattachmentIDPath,
+	}
+
+	operation.PathPattern = vs.pathPrefix + "getAttachment"
```

Comment:
its overwriting line no. 28 in same file. looks you missed it

Reply:
Addressed